docker-compose.yml

```diff
@@ -28,7 +28,8 @@ services:
     entrypoint: []
     command: bash -c "
         ([[ $$NETWORK == \"mainnet\" ]] && $$CMD --mainnet) ||
-        ($$CMD --testnet /config/*-$$NETWORK-byron-genesis.json)
+        ([[ $$NETWORK == \"alonzo-purple\" ]] && $$CMD --testnet /config/y88z6z1w0kwypj7zrlhk7vs6z8k1pckd-byron-genesis.json) ||
+        ($$CMD --testnet /config/kax0css4lx3ywihvsgrqjym0jpi20f99-byron-genesis.json)
```

Comment:
This will be very fragile because the hashes will change.

We really need the cardano-node docker image to provide an organised `/config` directory, like the one I have added into our docker image (new commits on this branch).

Reply:
Ok, how about we use config files from the cardano-wallet `/config` directory then?

Reply:
Yeah that would be good for the next release. But if anyone uses `docker-compose.yml` from master branch, there will be no `/config/${NETWORK}` in the `inputoutput/cardano-wallet:2021.8.11` image.

So perhaps we could use fragile nix store path hashes until the day of next release, then merge 50c19e8 with the release bump PR.

Reply:
Ok, I've added temp note to https://github.com/input-output-hk/cardano-wallet/wiki/Release-Checklist so we don't forget.